addons/progress-bar/addon.json

```diff
@@ -10,6 +10,10 @@
       "link": "https://scratch.mit.edu/users/jbthepig/"
     }
   ],
+  "latestUpdate": {
+    "version": "1.31.0",
+    "temporaryNotice": "The progress bar has had a redisign since Scratch Addons 1.30.1."
```

Comment:
Typo: "redisign"

Reply:
And add "v" before the version

Reply:
I don't think we normally even include the version in the notice tho. Maybe just exclude that part?

Reply:
@TheColaber Well, it's not bad to clarify that the change isn't technically new.

Reply:
> And add "v" before the version

Clarification: Keep `"version": "1.31.0"` the same but add the letter "v" in the `temporaryNotice` description.

Reply:
Should I remove "Scratch Addons" when I add the V?